composer.json

```
@@ -13,7 +13,8 @@
     ],
     "require": {
         "php": ">=5.6",
```

```suggestion
        "php": ">=7.1",
```

I wonder if this will break functional tests, though...